src/middlewared/middlewared/plugins/ups.py

```diff
@@ -31,6 +31,11 @@ class Config:
         data['mode'] = data['mode'].upper()
         data['shutdown'] = data['shutdown'].upper()
         data['toemail'] = [v for v in data['toemail'].split(';') if v]
+        if not data['shutdowncmd'] and not (
+            not (await self.middleware.call('system.is_freenas')) and (
+                await self.middleware.call('truenas.get_chassis_hardware')).startswith('TRUENAS-X')
+        ):
+            data['shutdowncmd'] = '/sbin/shutdown -p now'
```

Comment:
That may give the impression shutdown -p was entered by the user, and the UI will use it to populate the form, this block should be moved to the upsmon.conf instead, I think.

Reply:
But wouldn't that be kind of implicit ? User sees in the UI that there is no shutdown cmd default ( there is but he just doesn't know it yet ).

Reply:
We can have a tooltip in the UI or a placeholder, saying empty means shutdown -p.

If the user changes another value in the form it will also save the shutdown -p now command, which means if we ever want to change the default we wont be able to differentiate if it was entered by the user or its the system default.

Reply:
Makes sense, i'll have it updated asap :)